pandas/tests/indexes/test_common.py

```diff
@@ -468,8 +468,7 @@ def test_hasnans_isnans(self, index_flat):
 def test_sort_values_invalid_na_position(index_with_missing, na_position):
     with tm.maybe_produces_warning(
         PerformanceWarning,
-        pa_version_under7p0
-        and getattr(index_with_missing.dtype, "storage", "") == "pyarrow",
```

Comment:
How `pa_version_under7p0` was dropped here?

Reply:
Ah thanks. I was testing something locally and had committed this change by accident.